vars/republishDeployPipeline.groovy

```diff
@@ -36,14 +37,16 @@ def call(config) {
       }
     }
 
-    for (namespace in config.namespaces) {
-      stage("Helm Deploy: $namespace") {
-        helmDeploy([
-                releaseName      : config.application,
-                namespace        : namespace,
-                chartName        : config.chartNameOverride != null ? config.chartNameOverride : config.application,
-                imageTag         : params.imageTag
-        ])
+    if (config.skipDeploy != true) {
+      for (namespace in config.namespaces) {
```

Comment:
The doc comment now needs to say that namespaces is optional if skipDeploy is true